website/docs/script/rust/rust-quick-start.mdx

```diff
@@ -0,0 +1,574 @@
+---
+id: rust-quick-start
+title: "Quick Start"
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+import Link from "@docusaurus/Link";
+import Tooltip from "@components/Tooltip";
+import TutorialHeader from "@components/TutorialHeader";
+import ScriptTools from "./../_ScriptTools.mdx";
+
+# Quick Start
+
+<TutorialHeader
+  time={" 5 - 15min "}
+  tools={"script"}
+  customTools={[
+    <div key="debugger">
+      <Link
+        to="/docs/getting-started/installation-guide#5-ckb-debugger-01170"
+        target="_blank"
+        rel="noopener noreferrer"
+      >
+        CKB-Debugger (≥0.117.0)
+      </Link>
+    </div>,
+  ]}
+/>
+
+This chapter provides a step-by-step guide on developing contracts with Rust on CKB with several examples. As a "Quick Start" guide, it will not diving into in-depth details. For a more comprehensive understanding, please refer to the subsequent chapters.
+
+:::note
+Before proceeding, you should be familiar with:
+
+- CKB basics and the transaction structures
+- Rust
+- The `make` command
+  :::
+
+## Why Rust for CKB Contracts?
+
+Rust is widely used in developing core components of CKB due to its high performance and ease of use. Scripts often involve operations like signing, hashing, and handling large numbers. Rust’s superior performance helps reduce <Tooltip>Cycles</Tooltip> consumption. Furthermore, as Scripts are critical for asset management, Rust’s memory safety features help minimize vulnerabilities, making it a **recommended choice for Script development** .
```

Comment:
let's put this why-rust-for-ckb-contracts explaination to the https://docs.nervos.org/docs/script/program-language-for-script 's rust section.